### PR TITLE
New package: elvish

### PR DESCRIPTION
**Pull request by eunos-1128**
## Description

This pull request adds the [Elvish shell](https://elv.sh/) to Chromebrew. Elvish is a cross-platform, expressive, and versatile shell with features that support interactive use and scripting. It combines familiar shell syntax with advanced features like structured data, a powerful job control system, and a sophisticated command-line user interface.

##
Builds:
- [x] `x86_64`
- [ ] `i686`
- [ ] `armv7l`
##
Tested & Working properly:
- [x] `x86_64`
- [ ] `i686`
- [ ] `armv7l`  I don't have armv7l machines.
##
- [ ] This PR has no manifest .filelist changes. _(Package changes have neither added nor removed files.)_
##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/eunos-1128/chromebrew.git CREW_BRANCH=feature/elvish crew update \
&& yes | crew upgrade
```

### CODE REVIEW

**Review by satmandu — APPROVED** [review]

**Comment by uberhacker** [comment]
Thank you for the new package @eunos-1128!  I also noticed the elvish.filelist was empty and there are Rubocop suggestions.  Please address these issues and we can get this merged.

**Comment by satmandu** [comment]
In the root of your git clone, you can run `rubocop -r .rubocop.yml -A packages/elvish.rb` to automatically have rubocop try to fix those complaints...

**Comment by satmandu** [comment]
Also, once you have rubocop passing, we will have to generate new binaries.

**Comment by eunos-1128** [comment]
Hi, I fixed files you have reviewed.

File: tools/packages.yaml [hunk 1]
```diff
@@ -3869,6 +3874,7 @@ name: libbluray
 url: https://get.videolan.org/libbluray/
 activity: low
 ---
+
 ---
```
Comment (uberhacker) [inline tools/packages.yaml:3878]: Looks like we have an extra `---` line here.  We can fix on the next update.

File: tools/packages.yaml [hunk 2]
```diff
@@ -3869,7 +3874,6 @@ name: libbluray
 url: https://get.videolan.org/libbluray/
 activity: low
 ---
----
```
Comment (uberhacker) [inline tools/packages.yaml:3872]: I see you caught it!  Nice.

**Review by uberhacker — APPROVED** [review]

**Review by satmandu — APPROVED** [review]

**Pull request merged**